### PR TITLE
docs: Java restaurant deliveries

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

File: samples/grpc/restaurant-drone-deliveries-service-java/src/main/resources/persistence.conf [hunk 1]
```diff
@@ -23,9 +23,6 @@ akka {
         }
       }
 
-      # only time stamp based offsets used
-      offset-store.offset-table=""
```
Comment (patriknw) [inline samples/grpc/restaurant-drone-deliveries-service-java/src/main/resources/persistence.conf]: reason for removing? we only use timestamp offsets?

File: samples/grpc/restaurant-drone-deliveries-service-java/src/main/resources/application.conf [hunk 2]
```diff
@@ -0,0 +1,34 @@
+include "cluster"
+include "grpc"
+include "serialization"
+include "persistence"
+
+akka {
+  loglevel = DEBUG
+}
+
+akka.projection.grpc {
```
Comment (patriknw) [inline samples/grpc/restaurant-drone-deliveries-service-java/src/main/resources/application.conf:10]: shouldn't this be placed in grpc.conf?

File: samples/grpc/restaurant-drone-deliveries-service-java/src/main/resources/persistence.conf [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      }
+
+      state = {
+        additional-columns {
+          "CentralDrone" = ["central.drones.LocationColumn"]
+        }
+      }
+
+      # FIXME disabled until we have fixed handling of FilteredEvents for publishing in r2dbc
+      journal.publish-events = off
```
Comment (patriknw) [inline samples/grpc/restaurant-drone-deliveries-service-java/src/main/resources/persistence.conf:27]: is this still a problem? but maybe we'll go over all FIXME in a separate scan

Comment (johanandren) [reply samples/grpc/restaurant-drone-deliveries-service-java/src/main/resources/persistence.conf:27]: Yes, I'll do a follow up work through taking care of FIXMEs

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**